config/routes.rb

```diff
@@ -1,10 +1,15 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-  get '/users/:id', to: "users#show", as: :user
-  get '/users', to: "users#index", as: :users
-  get '/users/:user_id/posts/:id', to: "posts#show", as: :user_post
-  get '/users/:user_id/posts', to: 'posts#index', as: :user_posts
-  root to:'users#index'
+
   # Defines the root path route ("/")
   # root "articles#index"
+  root "users#index"
+  resources :users, only: [:index, :show]  do
+    resources :posts, only: [:index, :show] do
```

Comment:
- [ ] As the feature would allow users to create posts then there should be two routes for creating posts using `new` and `create` methods. Please ensure that the routes are added and that the app can create a new post